### PR TITLE
Smaller fixes

### PR DESCRIPTION
**Pull request by cstamas**
found during usual code reading.

### CODE REVIEW

File: nexus/nexus-proxy/src/main/java/org/sonatype/nexus/proxy/item/uid/RepositoryItemUidAttributeEventInspector.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public boolean accepts( Event<?> evt )
         final String simpleName = evt.getClass().getName();
 
         // TODO: nexus-proxy module does not reference plugin manager, so this is a quick'n'dirty workaround for now
-        return evt instanceof NexusStateChangeEvent
+        return evt instanceof NexusStartedEvent
```
Comment (peterlynch) [inline nexus/nexus-proxy/src/main/java/org/sonatype/nexus/proxy/item/uid/RepositoryItemUidAttributeEventInspector.java:37]: Why did this change?

This is the type of thing that should have a UT that self documents that this inspector responds only to _aaa_ events and does _xyz_ for such and such reason.

Comment (cstamas) [inline nexus/nexus-proxy/src/main/java/org/sonatype/nexus/proxy/item/uid/RepositoryItemUidAttributeEventInspector.java:37]: See commit message of corresponding commit:
44474dc405dceeb661abac26d9c93d517de3c773

**Comment by jdillon** [comment]
+something, okay one... though I don't understand the implications of the event change.

**Comment by cstamas** [comment]
Wrt event change: the event inspector was listening to superclass of "nexus state event changed" events. When this was implemented, it was fine, as it had only 2 extending classes (NexusStarted, NexusStopped).

But since then, we added 2 more: NexusInited and NexusStopping..... and it just doubled the invocation count on the component, while it always needed only one: NexusStarted (and plugin loaded and unloaded, but it's unrelated to this change).

In these 3 cases the UID attributes registry "updates" itself, as plugins might contribute attributes, and NexusStarted is needed to load the "core" attributes. That's all folks.

**Pull request merged**